### PR TITLE
Update product.tpl

### PR DESCRIPTION
**Pull request by Nobodaddy**
Added variable to suppress by default the conditions display

### CODE REVIEW

**Comment by Dh42** [comment]
Is there anything to control this?

**Comment by Nobodaddy** [comment]
Pardon? I don't understand your question. My proposed changes just provide an option to hide the conditions display on the product details page. There are no dependencies. 
Makes it easier for merchants who had to comment out directly the relevant code snippet in *Product.tpl* to reach the same result, until now.

**Comment by Traumflug** [comment]
This pull request makes more sense with the other three you provided at the same time, doesn't it?

If possible, it's usually a good idea to put related patches/commit into a single pull request. Github happily accepts PRs with 100 commits.

**Comment by Nobodaddy** [comment]
@Traumflug 
I totally agree, but although I made a lot of commits during the past years, I'm still a Github dummy. I simply do not know how to put these patches into a single pull request.

**Comment by Traumflug** [comment]
It works best by kicking out these Github "helper" tools out of the window :-) Using command-line `Git` and `Gitk --all` for viewing the results.

Anyways, the other patches are in a different repository anyways, so putting them into one PR isn't possible. But referencing them is: https://github.com/thirtybees/thirtybees/pull/260

File: product.tpl [hunk 1]
```diff
@@ -108,7 +108,8 @@
           <span class="editable" itemprop="sku"{if !empty($product->reference) && $product->reference} content="{$product->reference}"{/if}>{if !isset($groups)}{$product->reference|escape:'html':'UTF-8'}{/if}</span>
         </p>
 
-        {if !$product->is_virtual && $product->condition}
+	    	<!-- added variable to suppress by default the conditions display -->
```
Comment (Traumflug) [inline product.tpl]: It's unusual to put such comment into HTML. A diff like shown here nicely shows the change.

Comment (Nobodaddy) [reply product.tpl]: Yep! :)

**Review by firstred — APPROVED** [review]

**Comment by Traumflug** [comment]
Made a commit from the pieces here and pushed that: https://github.com/thirtybees/community-theme-default/commit/6dfd9d025c5c4b012a79287f56fff2d7a0bd5f11

Thank you very much!

**Comment by Dh42** [comment]
Thanks for your contribution @Nobodaddy

**Pull request closed without merging**